sdk/test-utils/test-utils/src/tracing/testSpan.ts

```diff
@@ -49,7 +49,7 @@ export class TestSpan implements Span {
   /**
    * Known attributes, if any.
    */
-  readonly attributes: SpanAttributes;
+  readonly attributes: Attributes;
```

Comment:
Was curious why the rename? FWIW I'm totally fine with it, just curious

Reply:
`SpanAttribute` is deprecated whereas `Attribute` is the new name.